core/frontend/vue.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
   transpileDependencies: ['vuetify', 'vuex-module-decorators'],
   pwa: {
     name: 'BlueOS',
-    themeColor: '#08c',
+    themeColor: '#2699D0',
```

Comment:
Should it have a comment pointing to the BR_BLUE color ? Should we define such colors in `.js` to allow the usage under both `.ts` and `.js` ?

Reply:
Maybe. I had some trouble importing them as `.ts`, but with `.js` I think there will be no problem.

Reply:
To make sure, you are checking that, right ?

Reply:
Completely forgot about that. Will take a look right now.

Reply:
More complicated than expected, as `vue.config.js` still runs on `commonjs`, but done.